tasks/transpile.js

```diff
@@ -155,11 +155,11 @@ module.exports = function (grunt) {
     function generateLocales(target, localeFiles, opts) {
         var files = localeFiles,
             code = [
-                'import moment from "./moment";',
+                'import moment from "./moment.js";',
                 'export default moment;'
             ].concat(files.map(function (file) {
                 var identifier = path.basename(file, '.js').replace('-', '_');
-                return 'import ' + identifier + ' from "./' + file + '";';
+                return 'import ' + identifier + ' from .js"./' + file + '";';
```

Comment:
The addition of the `.js` in this line appears to be the cause of the failing tests.

Reply:
yeah, i know...already fixed that locally, but there are still bugs...